src/ResponsiveAnalogRead.h

```diff
@@ -39,8 +39,11 @@ class ResponsiveAnalogRead
     //   increase this to lessen the amount of easing (such as 0.1) and make the responsive values more responsive
     //   but doing so may cause more noise to seep through if sleep is not enabled
     
+    ResponsiveAnalogRead();  //default constructor must be followed by call to begin function
```

Comment:
As mentioned, you might as well remove the parameter-less constructor from the .cpp file and make this line `ResponsiveAnalogRead() {}`

Reply:
OK, catching up with these suggested changes.
First, it totally makes sense to me to avoid duplicating code by calling begin() in the constructor that takes parameters (I think I was just trying to modify your code as minimally as possible).  So I'll fix that in a moment. 

I can see the sense in inlining the empty constructor, even if that wouldn't be my preference.  I don't personally agree with the suggestion to move the new one-liner constructor to the .h file... but it's just one of those things about personal preferences for code style.  I am happy to suit your preferences.  Updates in a moment one I've done enough testing to make sure I didn't introduce a typo.
Thanks!